### DISCUSSION TITLE
When is "Version 13" planned to release

### DISCUSSION THREAD

**Discussion by anbusampath** [graphql-java-kickstart/graphql-spring-boot#837] [Ideas]
When is "Version 13" planned to release? Because Graphql-java 18.x and Spring Boot 2.7.x released, Also these dependencies updated in main line. So version 13 would be great release to catch with latest dependencies.

**Comment by bsara** [comment]
or even v12.1.0 to follow along with the tools lib.

**Comment by bitbrain** [comment]
I'd recommend upgrading to Spring Boot 2.7 which comes with its own GraphQL starter now: https://spring.io/projects/spring-graphql

I already verified that it can do everything this starter does and it will be actively maintained as part of the Spring ecosystem.

**Reply by anbusampath** [reply]
> I already verified that it can do everything this starter does and it will be actively maintained as part of the Spring ecosystem.

Yeah, with new projects we can start with Spring for GraphQL. but we have project which is already using graphql-spring-boot for sometime. so it is easy to upgrade with this.

**Reply by bsara** [reply]
Spring for graphql has nothing on this library IMO. This library is way more intuitive and provides a much cleaner way of implementing resolvers that adheres to conventions that are more consistent with other graphql libraries and implementations that you'll find in other languages. Spring for Graphql also adds needless complexity, IMO.

**Reply by bitbrain** [reply]
To slightly disagree here, Spring GraphQL will be part of the Spring ecosystem, so simply upgrading Spring Boot to a newer version will ensure that Spring GraphQL gets upgraded too. The current version of graphql-java-kickstart is incompatible with Spring Boot 2.7 (at least for us) - also the Dataloader implementation of Spring GraphQL is way more flexible compared to this starter.

**Reply by bsara** [reply]
Does spring graphql not use graphql-java/java-dataloader? What flexibility is added?

**Reply by bitbrain** [reply]
In addition to the standard dataloader, it also provides its own "BatchLoader" implementation that is based on Flux/Mono (reactive streams)

**Comment by donbeave** [comment]
The same question from me as well, I'd like to release [graphql-java-datetime](https://github.com/tailrocks/graphql-java-datetime) the latest version, which provides scalars for the latest `graphql-java` library and Spring Boot starters for the Kickstart version of GraphQL Spring Boot. It will be nice if the `13.0.0` version will be released, as it already upgraded to graphql-java v18.

**Reply by oliemansm** [reply]
I'll make some time next week to release. Not sure if there are any blocking issues that will have to be fixed first.

**Reply by jimartinmoreno** [reply]
Yes, in our project are also looking forward for the next release to be available. It's there a list of candidates features to be released?

**Reply by donbeave** [reply]
Sorry for pinging here, how is progress with the release? Can I help with it?

**Reply by oliemansm** [reply]
There's a snapshot version available, but haven't been able to verify and regression test it yet. So any help in testing that snapshot version to check it works correctly would be helpful.

**Comment by oliemansm** [comment]
Finally got around to releasing v13. See also: https://github.com/graphql-java-kickstart/graphql-spring-boot/discussions/851, https://github.com/graphql-java-kickstart/graphql-spring-boot/releases/tag/v13.0.0 and https://github.com/graphql-java-kickstart/graphql-java-servlet/releases/tag/v13.0.0

**Comment by donbeave** [comment]
This is a great news on the weekend. Thank you so much!!!

Have a nice weekend!
